eng/ignore-links.txt

```diff
@@ -5,3 +5,4 @@ https://github.com/Azure/azure-sdk-for-net-pr/tree/feature/IoT-ADT/sdk/digitaltw
 https://github.com/Azure/azure-sdk-for-net-pr/blob/feature/IoT-ADT/sdk/iot/Azure.Iot.DigitalTwins/samples/DigitalTwinServiceClientSample/DTDL/Relationships/HospitalRelationships.json
 https://www.nuget.org/packages/Azure.ResourceManager.DigitalTwins
 https://contoso.azureedge.net/urlsigning/test?expires=2145916800&amp;keyid:key1&amp;signature=iTsrLX9rVAIJkSahBA_j5o9Azf5-j331ohxDR1Gx2js=
+https://docs.microsoft.com/en-us/azure/azure-resource-manager/management/azure-services-resource-providers
```

Comment:
Why is this added?

Reply:
That is a false positive that was flagged by the link checker. I added it here just to be able to test the PR.

Reply:
I think it was flagged because of "en-us" being in the link which is actually an issue we should fix.